app/views/govuk_publishing_components/components/_devolved_nations.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+
+  national_applicability ||= {}
+  applies_to ||= t('components.devolved_nations.applies_to')
+  applicable_nations = national_applicability.select { |k,v| v["applicable"] == true }.map { |k,v| t('components.devolved_nations.' + k + '') }.to_sentence(last_word_connector: ' and ')
+  nations_with_urls = national_applicability.select { |k,v| v["alternative_url"].present? }
+
+  heading_level ||= 2
+%>
+
+<% if national_applicability.any? { |k,v| v['applicable'] == true } %>
+  <%= tag.section class: "gem-c-devolved_nations" do %>
+    <%= content_tag(shared_helper.get_heading_level, class: "govuk-body govuk-!-margin-bottom-0") do %>
```

Comment:
For headings (in this case) we usually use the govuk-frontend class `govuk-heading-s`. `govuk-body` is normally reserved for `p` elements.

Reply:
Good to know. Thanks